seatunnel-connectors-v2/connector-tablestore/src/main/java/org/apache/seatunnel/connectors/seatunnel/tablestore/serialize/DefaultSeaTunnelRowSerializer.java

```diff
@@ -151,7 +156,8 @@ private PrimaryKeyColumn convertPrimaryKeyColumn(String columnName, Object value
             case BINARY:
                 return new PrimaryKeyColumn(columnName, PrimaryKeyValue.fromBinary((byte[]) value));
             default:
-                throw new UnsupportedOperationException("Unsupported primaryKeyType: " + primaryKeyType);
+                throw new TablestoreConnectorException(TablestoreConnectorErrorCode.UNSUPPORTED_PRIMARY_KEY_TYPE,
```

Comment:
CommonErrorCode.UNSUPPORTED_DATA_TYPE is better, please remove this redundant error code.